cis_v150/docs/cis_v150_5_3.md

```diff
@@ -6,14 +6,15 @@ Public access to remote server administration ports, such as 22 and 3389, increa
 
 ## Remediation
 
-Perform the following to implement the prescribed state:
+### From Console
```

Comment:
Can you please revert these changes as well? It is inconsistent with some of the other Remediation sections in these benchmarks, but since CIS has their documentation in this format, we choose to follow it as closely as we can.